modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/country/CommerceCountryManagerImpl.java

```diff
@@ -191,10 +191,16 @@ private GroupByStep _getGroupByStep(
 					channelFilterPredicate);
 
 				predicate = predicate.and(groupFilterPredicate);
-				predicate = predicate.and(
-					CountryTable.INSTANCE.billingAllowed.eq(billingAllowed));
-				predicate = predicate.and(
-					CountryTable.INSTANCE.shippingAllowed.eq(shippingAllowed));
+
+				if (billingAllowed) {
+					predicate = predicate.and(
```

Comment:
Change to following please:
```
				predicate = predicate.and(
						CountryTable.INSTANCE.billingAllowed.eq(billingAllowed));
```

Reply:
@andrea-ale-sbarra , could you double check this for me in terms of SF. Do we pass in the variable boolean or do we just set it to "true". I am not sure which one is the correct one actually in terms of SF.

Reply:
I think it's fine to use the variable in this case